src/Type.cc

```diff
@@ -699,6 +699,12 @@ TypePtr SetType::ShallowClone()
 
 SetType::~SetType() = default;
 
+FuncType::Capture::Capture(detail::IDPtr _id, bool _deep_copy)
+	: id(std::move(_id)), deep_copy(_deep_copy)
```

Comment:
Yeah, a fair question. I believe I'm the originator of the `arg_` style way back in the day, probably due to some overinfluential example in Stroustrup's original book :P. My druthers is the leading-`_` style because it's visually less cluttered, but I also appreciate that there are benefits to consistency. So I'll await a verdict from the Open Source Style Gods here and then am find changing if called for.

Reply:
I generally feel like having prefixes on function arguments doesn't gain anything. In the case of this function where you're moving the arguments directly, a compiler is smart enough to know the difference between the argument you're passing in and the variable you're moving it into.

Reply:
Does that extend to situations where you do the assignment in the body, like `id = id;`, rather than using initializers? I find with initializers, (1) the list gets visually unwieldy, (2) I get complaints about order-of-initialization in situations where the order clearly doesn't matter.

Reply:
I'm fine with moving this discussion to another venue in the interest of getting this in.

Reply:
@awelzel @timwoj I believe this can be closed out per the above